src/server/local/index.js

```diff
@@ -155,6 +155,13 @@ export default server => {
       const dataFilePath = path.join(process.cwd(), 'data', idx, 'index.json');
       sendDataFile(res, dataFilePath, next);
     })
+    .get('/static/js/comscore-1.0.js', (req, res, next) => {
+      const localComscorePath = path.join(
+        process.cwd(),
+        '/public/static/js/comscore-1.0.js',
```

Comment:
Apologies if this has been discussed while I was away, but how are we going to control the versions of the script? Are we just going to update this file and rename the script when Comscore releases a new change? 
I wonder if we should have something like a CHANGELOG where we track each script name explaining the changes.

Reply:
@DenisHdz @tochwill 

Thanks, both for highlighting this ^

I've added a readme/changelog to the same directory as the comscore script we are hosting. I've prefixed the comscore script with `main-` so that it follows the convention set in this regex: 
```(\\/static\\/js\\/(?:comscore\\/)?(main|vendor|${config[service].name})-\\w+\\.\\w+\\.js)```